src/content/overview/architecture.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Don't we already have this in https://github.com/giantswarm/docs/pull/2139/files? I assume this is a stub and I should only review the `introduction.md` file 😉.

Reply:
yes exactly (avoid failing the linting)